src/libaktualizr/storage/sqlstorage_test.cc

```diff
@@ -346,6 +363,72 @@ TEST(sqlstorage, DbMigration18to19) {
     FAIL() << "Too many rows";
   }
 }
+
+TEST(sqlstorage, DbMigration19to20) {
```

Comment:
What exactly is this test covering? Is it just making sure we don't erase things that were installed with version 19 after we migrate to version 20 since they don't have generation numbers?

Reply:
It's testing the migration with realistic data indeed. It gives a bit more confidence than just checking the table shapes, like schema_migration_test.sh does.